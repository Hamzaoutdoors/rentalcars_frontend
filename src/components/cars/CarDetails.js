const CarDetails = () => (
  <>
    Hello
  </>
);

export default CarDetails;
