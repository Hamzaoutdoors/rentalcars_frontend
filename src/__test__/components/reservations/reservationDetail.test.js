import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../../redux/configureStore';
import CarDetails from '../../../components/cars/CarDetails';

it('reservation detail component renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <CarDetails />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
