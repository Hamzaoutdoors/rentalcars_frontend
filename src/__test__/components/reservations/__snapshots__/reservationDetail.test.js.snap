// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reservation detail component renders correctly 1`] = `
<div
  className="sc-jrQzAO gURVnT"
/>
`;
